examples/ai/edge-functions/README.md

```diff
@@ -14,7 +14,7 @@ This demo consists of three parts:
 
 - Link your project: `supabase link`
 - Deploy Edge Functions: `supabase functions deploy`
-- Enable Database Webhooks in your [project dashboard](https://supabase.com/dashboard/project/_/database/hooks)
+- Update project config to enable webhooks: `supabase config push`
```

Comment:
**nitpick**

Maybe adding a link to the config reference: https://supabase.com/docs/guides/local-development/cli/config#experimental.webhooks.enabled